File: Backend/PlanningPoker.Domain/Common/Validation/Validation.Auth.cs
namespace PlanningPoker.Domain.Common.Validation;

public static partial class Validation
{
    public static class Auth
    {
        public static class Password
        {
            public const int minLength = 6;
            public const int maxLength = 50;
            public const string pattern =
                @"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]+$";
        }
    }
}

File: Client/src/validation/validation.auth.ts
export default {
    Password: {
        MinLength: 6,
        MaxLength: 50,
        Pattern:
            "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{6,50}$",
    },
};
